joplin/templates/wagtailadmin/pages/index.html

```diff
@@ -1,26 +0,0 @@
-{% extends "wagtailadmin/pages/index.html" %}
```

Comment:
Why is this deleted?

Reply:
The functionality of this file was added to the new `search_header.html`. It was showing up double or in unwanted places.